### PR TITLE
Adds a stamp sound effect when stamping papers

### PR DESCRIPTION
**Pull request by RedBaronFlyer**
## About The Pull Request

This pull request adds a .ogg file of a stamp stamping a paper, a playsound function for stamping a paper, stamping a paper while blind, and stamping yourself in the head to commit suicide.

In addition, it modifies the attributions.txt file to include credits to the creator of the stamp noise.

## Why It's Good For The Game

Most item interactions tend to have a sound effect that plays. Putting most items down, throwing them, cutting certain items with the knife, scanning bounty cubes, putting a destination tag on a package, etc. all play sounds, which makes it a bit odd that stamping papers doesn't. This PR adds audio feedback to stamping papers.

https://user-images.githubusercontent.com/45489195/210032424-d3168a16-3d4b-47d3-95f3-75a1d3a6aac9.mp4

## Changelog

Stamping paper now makes noise.

:cl:
soundadd: Added a new sound for stamping papers with a stamp. On an unrelated note, there are reports of stamp-based suicides becoming more noisy.
/:cl:

### CODE REVIEW

**Comment by RedBaronFlyer** [comment]
I'm leaving this as a draft for the moment since I want to make sure I got everything 100% correct before trying to do a full-on pull request.

**Comment by rellyreri** [comment]
I love you

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -407,6 +407,7 @@
 			add_stamp(writing_stats["stamp_class"], rand(0, 400), rand(0, 500), rand(0, 360), writing_stats["stamp_icon_state"])
 			user.visible_message(span_notice("[user] blindly stamps [src] with \the [attacking_item]!"))
 			to_chat(user, span_notice("You stamp [src] with \the [attacking_item] the best you can!"))
+			playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, TRUE)
```
Comment (Mothblocks) [inline code/modules/paperwork/paper.dm:410]:
```suggestion
			playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, vary = TRUE)
```

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -540,6 +541,7 @@
 
 			add_stamp(stamp_class, stamp_x, stamp_y, stamp_rotation, stamp_icon_state)
 			user.visible_message(span_notice("[user] stamps [src] with \the [holding.name]!"), span_notice("You stamp [src] with \the [holding.name]!"))
+			playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, TRUE)
```
Comment (Mothblocks) [inline code/modules/paperwork/paper.dm:544]:
```suggestion
			playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, vary = TRUE)
```

File: code/modules/paperwork/stamps.dm [hunk 3]
```diff
@@ -15,6 +15,7 @@
 
 /obj/item/stamp/suicide_act(mob/living/user)
 	user.visible_message(span_suicide("[user] stamps 'VOID' on [user.p_their()] forehead, then promptly falls over, dead."))
+	playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, TRUE)
```
Comment (Mothblocks) [inline code/modules/paperwork/stamps.dm:18]:
```suggestion
	playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, vary = TRUE)
```

**Review by Mothblocks — APPROVED** [review]

**Comment by RedBaronFlyer** [comment]
Committed changes as requested. Is there any difference between TRUE and vary = TRUE? It seemed like they did the same thing regarding sound variation when an audio file is played?

Unless that isn't what that does at all and I'm just silly.

**Comment by Mothblocks** [comment]
[Prefer named arguments when the meaning is not obvious](https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#prefer-named-arguments-when-the-meaning-is-not-obvious)

Undraft if ready BTW

**Pull request merged**